### ISSUE TITLE
Can not re-open a photo

### ISSUE THREAD

**Issue by GeoHigh125** [geopaparazzi/geopaparazzi#629]
In the new installed version 6.0.0RC. I can not reopen the created pictures in a form

**Comment by moovida** [comment]
Yes, this has never been possible. We can leave it open as feature request.
